### PR TITLE
add conflict last n years

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

File: openreview/tools.py [hunk 1]
```diff
@@ -1628,7 +1628,7 @@ def get_conflicts(author_profiles, user_profile, policy='default'):
         author_relations.update(author_info['relations'])
```
Comment (nlbahy) [inline openreview/tools.py:1628]: Does it not need to pass n_years when getting the author info?

Comment (nlbahy) [reply openreview/tools.py:1628]: I meant to link the comment to this line: author_info = info_function(profile)

File: openreview/tools.py [hunk 2]
```diff
@@ -1628,7 +1628,7 @@ def get_conflicts(author_profiles, user_profile, policy='default'):
...
         author_publications.update(author_info['publications'])
 
-    user_info = info_function(user_profile)
+    user_info = info_function(user_profile, n_years)
```
Comment (melisabok) [inline openreview/tools.py:1631]: i'm passing the parameter here

File: openreview/tools.py (hunk 1 above)
Comment (melisabok) [reply openreview/tools.py:1628]: oops you are right!
I missed this part.

**Review by nlbahy — APPROVED** [review]

**Pull request merged**